frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -1,53 +1,61 @@
+import { execSync } from 'child_process';
 import * as _ from 'lodash';
 import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { getAnnotations, getLabels } from '../../src/selectors/selectors';
 import {
   removeLeakedResources,
   withResource,
   createResources,
   deleteResources,
 } from '@console/shared/src/test-utils/utils';
-import { getAnnotations, getLabels } from '../../src/selectors/selectors';
-import {
-  getResourceObject,
-  resolveStorageDataAttribute,
-  selectNonDefaultAccessMode,
-  selectNonDefaultVolumeMode,
-} from './utils/utils';
-import { Wizard } from './models/wizard';
 import {
   VM_BOOTUP_TIMEOUT_SECS,
   CLONE_VM_TIMEOUT_SECS,
-  VM_ACTION,
-  CLONED_VM_BOOTUP_TIMEOUT_SECS,
-  VM_STATUS,
-  commonTemplateVersion,
   COMMON_TEMPLATES_REVISION,
-  DISK_INTERFACE,
+  commonTemplateVersion,
   STORAGE_CLASS,
-} from './utils/consts';
-import { multusNAD, cdGuestTools, basicVMConfig } from './utils/mocks';
+  CLONED_VM_BOOTUP_TIMEOUT_SECS,
+} from './utils/constants/common';
 import {
-  vmConfig,
-  getProvisionConfigs,
+  multusNAD,
+  cdGuestTools,
+  flavorConfigs,
+  hddDisk,
+  provisionSources,
+  rootDisk,
   getTestDataVolume,
-  VMTestCaseIDs,
   kubevirtStorage,
-} from './vm.wizard.configs';
+  getDiskToCloneFrom,
+} from './mocks/mocks';
+import { ProvisionSource, Workload, OperatingSystem, OSIDLookup } from './utils/constants/wizard';
+import { vmPresets, basicVM } from './mocks/vmBuilderPresets';
+import { VMBuilder } from './models/vmBuilder';
 import {
-  Flavor,
-  OperatingSystem,
-  OSIDLookup,
-  ProvisionConfigName,
-  WorkloadProfile,
-} from './utils/constants/wizard';
+  resolveStorageDataAttribute,
+  selectNonDefaultAccessMode,
+  selectNonDefaultVolumeMode,
+} from './utils/utils';
+import { DISK_DRIVE, DISK_INTERFACE, VM_STATUS } from './utils/constants/vm';
 
 describe('Kubevirt create VM using wizard', () => {
   const leakedResources = new Set<string>();
-  const provisionConfigs = getProvisionConfigs();
   const testDataVolume = getTestDataVolume();
   const defaultAccessMode = resolveStorageDataAttribute(kubevirtStorage, 'accessMode');
   const defaultVolumeMode = resolveStorageDataAttribute(kubevirtStorage, 'volumeMode');
-  const wizard = new Wizard();
+
+  const getDataVolumeByPrefix = (prefix: string) => {
+    const dvs = JSON.parse(
+      execSync(`kubectl get dv -n ${testName} -ojson | jq '.items'`).toString(),
+    );
+    return _.find(dvs, (dv) => dv.metadata.name.includes(prefix));
+  };
+
+  const VMTestCaseIDs = {
```

Comment:
wouldn't it  make more sense if it was maping of `testID -> presetInstance`?

Reply:
It would actually

Reply:
updated